src/algorithm.rs

```diff
@@ -156,6 +167,39 @@ impl Bft {
         }
     }
 
+    #[cfg(feature = "verify_req")]
+    fn initialize(
+        s: Sender<BftMsg>,
+        r: Receiver<BftMsg>,
+        ts: Sender<TimeoutInfo>,
+        tn: Receiver<TimeoutInfo>,
+        local_address: Target,
+    ) -> Self {
+        info!("BFT State Machine Launched.");
+        Bft {
+            msg_sender: s,
+            msg_receiver: r,
+            timer_seter: ts,
+            timer_notity: tn,
+
+            height: INIT_HEIGHT,
+            round: INIT_ROUND,
+            step: Step::default(),
+            feed: None,
+            proposal: None,
+            votes: VoteCollector::new(),
+            lock_status: None,
+            last_commit_round: None,
+            last_commit_proposal: None,
+            authority_list: Vec::new(),
+            htime: Instant::now(),
+            height_filter: HashMap::new(),
+            round_filter: HashMap::new(),
+            params: BftParams::new(local_address),
+            verify_result: HashMap::new(),
```

Comment:
Why not use #[cfg(feature = "verify_req")] to mark only this line

Reply:
If that, I think it will be inaesthetic.